### ISSUE TITLE
updating

### ISSUE THREAD

**Issue by ftrescue** [datarhei/restreamer-ui#7]
Hi guy hope you can help i removed the container and then added the code 
docker run -d --restart=always --name restreamer \
-v /opt/restreamer/config:/core/config \
-v /opt/restreamer/data:/core/data \
-p 8080:8080 -p 8181:8181 -p 1935:1935 \
datarhei/restreamer:latest

but it wont update the program as still says need a update any i dears would help thanks

if you need to logon i do have anydesk  thanks

**Comment by jstabenow** [comment]
Hey @ftrescue 
to update your local image exec `docker pull datarhei/restreamer:latest`.

**Comment by ftrescue** [comment]
> Hey @ftrescue to update your local image exec `docker pull datarhei/restreamer:latest`.

thats in the code i posted ?

**Comment by jstabenow** [comment]
No, `docker run ...` only loads the image if unavailable locally. 
And `docker pull` updates an existing image.

Please run:
```sh
$ docker pull datarhei/restreamer:latest
$ docker run -d --restart=always --name restreamer \
   -v /opt/restreamer/config:/core/config \
   -v /opt/restreamer/data:/core/data \
   -p 8080:8080 -p 8181:8181 -p 1935:1935 \
   datarhei/restreamer:latest
